### ISSUE TITLE
ComputeStructureFromKnownPoses Segmentation Fault Core Dumped

### ISSUE THREAD

**Issue by Yangyp29** [openMVG/openMVG#2375]
Hi, I am running the pipeline on different datasets. In some, it shows Segmentation Fault (Core Dumped) in Compute pairwise fundamental guided matching of ComputeStructureFromKnownPoses.
Can you help me out, why is it so in some of the datasets. What might be the possible defect in Dataset?
Thanks in advance! :)

st@st-vm:~/yyp/openMVG/openMVG_Build/software/SfM$ **export OMP_NUM_THREADS=1**
st@st-vm:~/yyp/openMVG/openMVG_Build/software/SfM$ $sfm_bin/openMVG_main_ComputeStructureFromKnownPoses -i /home/st/yyp/JointCalibration/test_ours/pyOut/sfm_data.bin -o /home/st/yyp/JointCalibration/test_ours/pyOut/robustFitting.json -m /home/st/yyp/JointCalibration/test_ours/pyOut/
INFO: [main_ComputeStructureFromKnownPoses.cpp:56] Compute Structure from the provided poses
INFO: [main_ComputeStructureFromKnownPoses.cpp:134] Loaded a sfm_data scene with:
 #views: 12
 #poses: 10
 #intrinsics: 12
 #tracks: 0
INFO: [loggerprogress.hpp:79] [- Regions Loading -] 50%
INFO: [loggerprogress.hpp:79] [- Regions Loading -] 100%
INFO: [main_ComputeStructureFromKnownPoses.cpp:299] Robust triangulation of the tracks:  [Triangulation of guided epipolar geometry matches]
INFO: [loggerprogress.hpp:79] [Computing frustum intersection] 20%
INFO: [loggerprogress.hpp:79] [Computing frustum intersection] 40%
INFO: [loggerprogress.hpp:79] [Computing frustum intersection] 60%
INFO: [loggerprogress.hpp:79] [Computing frustum intersection] 80%
INFO: [loggerprogress.hpp:79] [Computing frustum intersection] 100%
INFO: [loggerprogress.hpp:79] [Pairwise fundamental guided matching] 20%
INFO: [loggerprogress.hpp:79] [Pairwise fundamental guided matching] 40%
**Segmentation fault (core dumped)**

**Comment by pmoulon** [comment]
Would you be able/willing to add some debug log in the code?

Or share your dataset (not image but just files required for this binary to run?

**Comment by Yangyp29** [comment]
> Would you be able/willing to add some debug log in the code?
> 
> Or share your dataset (not image but just files required for this binary to run?

Do you mean sfm_data.bin？

**Comment by Yangyp29** [comment]
> Would you be able/willing to add some debug log in the code?
> 
> Or share your dataset (not image but just files required for this binary to run?

Do you mean sfm_data.bin？

**Comment by pmoulon** [comment]
Yes I would need the features, sfm_data.bin and the files you gave as input to the binary above

**Comment by pmoulon** [comment]
Any update on this, or data you can share?
